test/k8s-integration/driver.go

```diff
@@ -65,12 +67,12 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 
 	// TODO (#139): wait for driver to be running
 	time.Sleep(time.Minute)
-	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", "default")
-	err = runCommand("Checking driver pods", statusCmd)
+	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", "gce-pd-csi-driver")
+	out, err = statusCmd.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("failed to check driver pods: %v", err)
+		return fmt.Errorf("failed to get nodes: %v", err)
 	}
-
+	klog.V(2).Infof("GCE PD driver pods: \n %v", string(out))
```

Comment:
BTW, this is fixed in https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/140

Reply:
ack.